### PR TITLE
Ammonia servo

### PR DESCRIPTION
**Pull request by cgiger00**
builds with jarvis, has not been tested, although since the code was mostly changing constants from the previous servo code it'll probably work

### CODE REVIEW

File: beaglebone/servo/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+﻿Code for Servos
+ ---
+### About
+
+This code is intended to run the servos in order to lower the beaker for science testing. The beaker should stay in place until an empty LCM struct is sent where it will lower the beaker.
+
+### LCM Channels Publishing/Subscribed To
+**ServoCMD**[subscriber] \
+Messages:  [ServoCMD.lcm](https://github.com/jjtom34/mrover-workspace/blob/AmmoniaServo/rover_msgs/ServoCmd.lcm) "/servocmd" \
```
Comment (raytitan) [inline beaglebone/servo/README.md:9]: "/servo_cmd" right?

File: beaglebone/servo/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+### Usage
+#### Require Electronic Components:
+- 1 Servo
+- 1 Beaglebone Black board or similar board
+
+
+#### Setup
+When the servo is set up with the beaker, the program should be run as follows
+`$ ~/cd mrover-workspace`
+`$ /jarvis build beaglebone/ammoniaservo` \
```
Comment (raytitan) [inline beaglebone/servo/README.md:23]: why wasn't this named "servo" or "science_servo"?

File: beaglebone/servo/src/__main__.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import time
+import Adafruit_BBIO.PWM as PWM
+import lcm
+from rover_msgs import ServoCmd
+
+
+# Don't know if this is the same for the servo we're using
+SERVO_MAX_DC = 10.0
+SERVO_MIN_DC = 4.0
```
Comment (raytitan) [inline beaglebone/servo/src/__main__.py:9]: Have these values been tested with the science box servos?

File: beaglebone/servo/src/__main__.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    dc = angle_to_dc(degrees)
+    PWM.start(pin, dc, 50)
+
+
+def servocmd_callback(channel, msg):
+    servo = ServoCmd.decode(msg)
+    # Blue callback
+    if (servo.id == "amino_blue"):
+        run_servo(amino_blue, servo.position)
+        time.sleep(4)
```
Comment (raytitan) [inline beaglebone/servo/src/__main__.py:44]: what's with these sleeps?

Comment (cgiger00) [reply beaglebone/servo/src/__main__.py:44]: idk jonathan put them in, i can get rid of them i guess?

File: beaglebone/servo/src/__main__.py (hunk 3 above)
Comment (cgiger00) [reply beaglebone/servo/src/__main__.py:9]: this has not been tested...values from last year's code ISH has been looking for the servos

File: beaglebone/servo/README.md (hunk 2 above)
Comment (cgiger00) [reply beaglebone/servo/README.md:23]: it's servo in the code i hadn't updated the readme

File: beaglebone/servo/README.md (hunk 1 above)
Comment (cgiger00) [reply beaglebone/servo/README.md:9]: right

**Review by raytitan — APPROVED** [review]

**Pull request merged**